src/components/CaptureDetailDialog.js

```diff
@@ -82,11 +94,18 @@ function CaptureDetailDialog(props) {
   // console.log('render: capture detail dialog');
   const { open, capture } = props;
   const cdContext = useContext(CaptureDetailContext);
+  const appContext = useContext(AppContext);
+  const hasApproveTreePermission = hasPermission(appContext.user, [
+    POLICIES.APPROVE_TREE,
```

Comment:
Need to include `POLICIES.SUPER_PERMISSION`

Reply:
I Have added this permission but the `onCaptureTagDelete` function is still undefined on the props.